mods/ra/weapons/explosions.yaml

```diff
@@ -212,6 +212,14 @@ UnitExplodeSmall:
 		SmudgeType: Crater
 		InvalidTargets: Structure, Wall, Trees
 
+CivBuildingExplosion:
```

Comment:
Otherwise the civilians won't panic, but just stand still (on top of each other, if multiple spawned).